src/flb_upstream.c

```diff
@@ -65,6 +65,25 @@ struct flb_config_map upstream_net[] = {
      "before it is retired."
     },
 
+    {
+     FLB_CONFIG_MAP_INT, "net.tcp_keepalive_time", "0",
```

Comment:
I think default values should be avoided since every system can have an implicit default that might not match this ```0```. I am thinking mostly on current users that my face a different internal setup due to this change.

Reply:
Replaced with -1, and as indicated in other comment, the system-wide default will be then used if the configuration does not specify a desired value